src/sentry/api/endpoints/group_details.py

```diff
@@ -208,11 +208,18 @@ def get(self, request, group):
             get_range = lambda model, keys, start, end, **kwargs: \
                 {k: tsdb.make_series(0, start, end) for k in keys}
             tags = []
+            user_reports = UserReport.objects.none()
+
         else:
             get_range = functools.partial(tsdb.get_range, environment_id=environment_id)
             tags = tagstore.get_group_tag_keys(
                 group.project_id, group.id, environment_id, limit=100)
 
+            if environment_id is None:
+                user_reports = UserReport.objects.filter(group=group)
+            else:
+                user_reports = UserReport.objects.filter(group=group, environment_id=environment_id)
```

Comment:
This is just a style thing, but you can actually do this instead:
```python
user_reports = UserReport.objects.filter(group=group)
if environment_id is not None:
     user_reports = user_reports.filter(environment_id=environment_id)
```

Reply:
Also a super small nit is that the newline on L217 is a breaks up the else in an odd place visually.

Reply:
Removed the newline

Gonna leave the user_reports definition because symmetric if/else looks better to me and it's used elsewhere in this function